src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1512,19 +1512,25 @@ public void TestTimesStringV21()
         [Test]
         public void TestAppend()
         {
-            Hash assigns = Hash.FromAnonymousObject(new { a = "bc", b = "d" });
+            Hash assigns = Hash.FromAnonymousObject(new { a = "bc", b = "d", d = "x" });
             Helper.AssertTemplateResult(expected: "bcd", template: "{{ a | append: 'd'}}", localVariables: assigns);
             Helper.AssertTemplateResult(expected: "bcd", template: "{{ a | append: b}}", localVariables: assigns);
+            Helper.AssertTemplateResult(expected: "bcbc", template: "{{ a | append: a}}", localVariables: assigns);
             Helper.AssertTemplateResult(expected: "/my/fancy/url.html", template: "{{ '/my/fancy/url' | append: '.html' }}");
             Helper.AssertTemplateResult(expected: "website.com/index.html", template: "{% assign filename = '/index.html' %}{{ 'website.com' | append: filename }}");
+            Helper.AssertTemplateResult(expected: "hi", template: "{{ nonesuch | append: 'hi' }}");
+            Helper.AssertTemplateResult(expected: "hi", template: "{{ 'hi'| append: nonesuch }}");
         }
 
         [Test]
         public void TestPrepend()
         {
             Hash assigns = Hash.FromAnonymousObject(new { a = "bc", b = "a" });
-            Helper.AssertTemplateResult("abc", "{{ a | prepend: 'a'}}", assigns);
-            Helper.AssertTemplateResult("abc", "{{ a | prepend: b}}", assigns);
+            Helper.AssertTemplateResult(expected: "abc", template: "{{ a | prepend: 'a'}}", localVariables: assigns);
+            Helper.AssertTemplateResult(expected: "abc", template: "{{ a | prepend: b}}", localVariables: assigns);
+            Helper.AssertTemplateResult(expected: "bcbc", template: "{{ a | prepend: a}}", localVariables: assigns);

```

Comment:
nit: this test doesn't add any more than the previous

Reply:
This is testing that the lhs and rhs can contain the same argument. I thought that was a fairly common edge case to test. I'm happy to remove it if you think it's unnecessary.